src/registry/textarea/edit-validation.ts

```diff
@@ -0,0 +1,31 @@
+import {IntlShape} from 'react-intl';
+import {z} from 'zod';
+
+import {buildCommonSchema} from '@/registry/validation';
+
+const textareaSchema = z.string().optional();
+
+// case for when component.multiple=false
+const singleValueSchema = z
+  .object({multiple: z.literal(false)})
+  .and(z.object({defaultValue: textareaSchema}));
+
+// case for when component.multiple=true
+const multipleValueSchema = z
+  .object({multiple: z.literal(true)})
+  .and(z.object({defaultValue: textareaSchema.array()}));
+
+// Omit `autoExpand` as it will be set in the UI via a checkbox
+const textareaSpecific = z.object({
+  validate: z.object({
+    maxLength: z.number().int().gte(1).optional(),
+  }),
+  rows: z.number().int().gte(1),
```

Comment:
this can also be optional, no?

Reply:
It is marked as required in types :shrug:

Reply:
Yeah I see formio itself sets a default, but also supports falsy values: https://github.com/formio/formio.js/blob/4.13.x/src/components/textarea/TextArea.js#L53

I think we should make it optional in the types indeed.